package.json

```diff
@@ -6,7 +6,6 @@
   "scripts": {
     "lint": "standard | snazzy",
     "unit": "tap test.js",
```

Comment:
```suggestion
    "unit": "tap -J test.js",
```

Reply:
There is no need to `-J` the new version of `tap` does that by default :)